lib/errors/connection/connection-closed-error.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const BaseError = require('../base-error');
+
+/**
+ * Thrown when a connection to a database is closed while an operation is in progress
+ */
+class ConnectionClosedError extends BaseError {
+  constructor(message) {
+    super(message);
+    this.name = 'SequelizeConnectionClosedError';
+    Error.captureStackTrace(this, this.constructor);
```

Comment:
Again, this is redundant with node 10+

Reply:
It's in the other error classes so maybe we should just wait for another PR to remove them all?

Reply:
All I'm saying there isn't really a need to add more, feel free you create a PR for the others though :)

Reply:
cool, I left it out when refactoring the error to what Sushant requested